coursebuilder/views/unit.html

```diff
@@ -126,7 +126,11 @@
                 {% if next_button_url %}
                   <a href="{{ next_button_url }}" class="btn right">{{ gettext('Next') }}</a>
                 {% else %}
-                  <a href="course" class="btn right"> {{ gettext('End') }} </a>
+                  {% if unit.title in course_outline[((course_outline|length) - 1)].title %}
```

Comment:
@Tawakalt I like that you are now looking into courser_outline. I just however broke the feature by rearranging the units [a feature in the GCB] https://tawakalt-dot-cfafrica-mooc-dev.appspot.com/test/dashboard?action=outline what is the default ordering and is it possible to have two units with the same title?

Reply:
@chege Having the same title is possible and will break the feature, but I can fix that by using the id instead. I however don't see how I can fix it when the courses are being rearranged.